docs/config.rst

```diff
@@ -51,6 +51,13 @@ Configuration Options
   * Default: "localhost"
   * Importance: high
 
+``kafkastore.security.protocol``
```

Comment:
Can you please also update https://github.com/confluentinc/docs/blob/master/kafka/platform-security.rst once we merge this?